### PR TITLE
{chem}[gfbf/2022b,iimkl/2022b] dftd4 v3.4.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@sassy-crick could you update the PR with a list of its dependent PRs?

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/d/dftd4/dftd4-3.4.0-iimkl-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+github_account = 'dftd4'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['f3b0a16a952817ae48e819626e13676fba3b61c8beea47b0f8ada2fbb679fb7b']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+    ('binutils', '2.39'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/d/dftd4/dftd4-3.4.0-iimkl-2022b.eb:21]:
```suggestion
```

File: easybuild/easyconfigs/d/dftd4/dftd4-3.4.0-gfbf-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+github_account = 'dftd4'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['f3b0a16a952817ae48e819626e13676fba3b61c8beea47b0f8ada2fbb679fb7b']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+    ('binutils', '2.39'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/d/dftd4/dftd4-3.4.0-gfbf-2022b.eb:21]:
```suggestion
```

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**